client/src/pages/About.js

```diff
@@ -1,12 +1,21 @@
+import { Link } from "react-router-dom";
+import { Button } from "react-bootstrap";
 const About = () => (
-	<main role="main">
+	<main role="main" style={{ width:"800px",margin:"100px auto" }}>
 		<div>
 			<h1>About</h1>
 			<p>
 				Starter kit for full-stack JavaScript projects. For more information,
 				see the wiki:
 			</p>
-			<a href="https://github.com/textbook/starter-kit/wiki">Wiki</a>
+			
+			<Link to="/login/this/site">
+				<Button variant="primary" style={{ width:"150px",margin:"20px",backgroundColor:"#0F172A",border:"none",height:"50px"}}>Log In</Button>
+			</Link>
+
+			<Link to="/registraction/this/site">
```

Comment:
Hi Christian, not sure if this is on the same thing Charles commented on but please amend the spelling of registraction to registration